src/_guides/language/language-tour.md

```diff
@@ -3281,6 +3285,32 @@ For more information, see the informal
 [noSuchMethod forwarding specification.](https://github.com/dart-lang/sdk/blob/master/docs/language/informal/nosuchmethod-forwarding.md)
 
 
+### Extension methods
+
+Extension methods, introduced in Dart 2.7,
+are a way to add functionality to existing libraries.
+You might not ever need to _use_ extension methods, much less _create_ them.
+But if you generate code or find yourself writing lots of wrappers,
+consider using extension methods to reduce
+the amount of boilerplate code that you need to write.
+
+You might use extension methods without even knowing it.
```

Comment:
Here in the tour, I'd move this below the example.

Reply:
I rearranged it a bit.